src/backend/actual.h

```diff
@@ -53,8 +53,8 @@ class TensorActual : public Actual {
 
 class SparseTensorActual : public TensorActual {
 public:
-  SparseTensorActual(const unsigned* coords, const unsigned* sinks, void* data)
```

Comment:
Can we change to the following idiom since we use it in the rest of Simit?:
```
SparseTensorActual(const unsigned* coords, const unsigned* sinks, void* data)
    : TensorActual(data), coords{coords}, sinks{sinks} {}
```